packages/postgres-database/src/simcore_postgres_database/models/projects_to_jobs.py

```diff
@@ -0,0 +1,43 @@
+import sqlalchemy as sa
+
+from ._common import RefActions
+from .base import metadata
+from .projects import projects
+
+projects_to_jobs = sa.Table(
+    # Maps projects used as jobs in the public-api
+    "projects_to_jobs",
+    metadata,
+    sa.Column(
+        "id",
+        sa.BigInteger,
+        primary_key=True,
+        autoincrement=True,
+        doc="Identifier index",
+    ),
```

Comment:
I woudl like to ask you to change this to a UUID or soemthing else but not an integer. Maybe you can also have a primary key composed by more keys.
We are trying to remove these in the upcoming merge of the platforms.

I think you can just use the project_uuid as a primary key as well in this case.

Reply:
i would like to keep another id for the association. I can turn it into a UUID if you think it is better 
The uniqueness condition you refer is already in place

Reply:
based on your feedback and since i am not really using this id. I decided to remove it. For the moment the unique composite (project-uuid, parent_resource) defines uniquely the rows.